packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -84,11 +84,102 @@ const testDatas = {
   ...validatorTestData,
 };
 
+const filteredOperations: FilteredOperation[] = [
+  /*
+   #5693
+   missing finalized
+   */
+  {id: "getStateRoot", requiredResponseProperties: ["finalized"]},
```

Comment:
This reads wrong to me, what about

```suggestion
  {id: "getStateRoot", ignoredResponseProperties: ["finalized"]},
```

could also have a single array with "ignore objects", e.g.

```suggestion
  {id: "getStateRoot", ignoredProperties: [{response: "finalized"}]},
```

but this probably makes things more complicated